server/src/main/java/org/apache/iotdb/db/engine/compaction/inner/sizetiered/SizeTieredCompactionRecoverTask.java

```diff
@@ -65,13 +67,21 @@ public SizeTieredCompactionRecoverTask(
   }
 
   /**
-   * Clear unfinished compaction task, there are several situations:
+   * We support tmpTargetFile is xxx.target, tmpTargetResourceFile is xxx.target.resource,
+   * targetFile is xxx.tsfile, targetResourceFile is xxx.tsfile.resource. To clear unfinished
+   * compaction task, there are several situations:
    *
    * <ol>
-   *   <li><b>Target file is uncompleted</b>: delete the target file and compaction log.
-   *   <li><b>Target file is completed, not all source files have been deleted</b>: delete the
-   *       source files and compaction logs
-   *   <li><b>Target file is completed, all source files have been deleted, compaction log file
+   *   <li>Compaction log is incomplete, then delete it and return.
+   *   <li>TmpTargetFile exists, then delete targetFile if exists and remove tmpTargetFile to
```

Comment:
"remove sth to sth" is not a legal English grammar. What if tmpTargetFile is not complete, we should remove TmpTargetFile and its resource then,